CHANGELOG.md

```diff
@@ -122,14 +133,30 @@
 * [ENHANCEMENT] Documented how to configure HA deduplication using Consul in a Mimir Helm deployment. #2972
 * [ENHANCEMENT] Improve `MimirQuerierAutoscalerNotActive` runbook. #3186
 * [ENHANCEMENT] Improve `MimirSchedulerQueriesStuck` runbook to reflect debug steps with querier auto-scaling enabled. #3223
+* [ENHANCEMENT] Mimir 2.4 release notes. #3104 #3319
```

Comment:
I would remove this. It was part of 2.4 release.

Reply:
Fixed: 9403c0e75d32c45e6ae446e2f11e20ce168ebb45